lib/discourse_voting/topic_extension.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module DiscourseVoting
+  module TopicExtension
+    def self.prepended(base)
+      base.has_one :vote_counter, class_name: 'DiscourseVoting::VoteCounter'
+      base.has_many :votes, class_name: 'DiscourseVoting::Vote'
```

Comment:
Should we destroy the `votes` and `vote_counter` records if the topic is destroyed?